app/controllers/settings/password.js

```diff
@@ -16,8 +16,8 @@ export default Controller.extend({
           this.get('notify').success(this.get('l10n').t('Password updated successfully'));
         })
         .catch(error => {
-          if (error.error) {
-            this.get('notify').error(this.get('l10n').t(error.error));
+          if (error.errors) {
```

Comment:
@Anupam-dagar It may be possible that error is because of reasons other than `wrong current password` e.g. `server not responding` etc. How do you plan to catch those errors and display them correctly.

Reply:
@pradeepgangwar I guess then it's correct only to show the error received by the server and on the server side we can make this error better. I wrote the latter part just to justify it more clearly.
Should I make a PR on the server side to improve the error message?
I'll update my PR with only showing the error received from the server.

Reply:
Yes, it looks better to do this on server and fetch error from there.

Reply:
I have done the required frontend changes, will open a server PR next.